ui/components/multichain/account-details/account-details.js

```diff
@@ -27,11 +27,11 @@ import {
   AvatarAccountVariant,
   Box,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../component-library';
+import { ModalContent } from '../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../component-library/modal-header/deprecated';
 import { AddressCopyButton } from '../address-copy-button';
 import { AccountDetailsAuthenticate } from './account-details-authenticate';
 import { AccountDetailsDisplay } from './account-details-display';
```

Comment:
Screencasts showing no visual changes to UI

### Before

https://github.com/MetaMask/metamask-extension/assets/8112138/62e74694-e56f-4cba-b97c-4ae4a95ec3d7

### After

https://github.com/MetaMask/metamask-extension/assets/8112138/ba00c120-6deb-44ab-ad76-6793e908e03b